### PR TITLE
Doc adjustments for occ trashbin:restore

### PR DESCRIPTION
**Pull request by R0Wi**
See https://github.com/nextcloud/server/pull/39818

- [ ] Wait for https://github.com/nextcloud/server/pull/39818 to be merged

### CODE REVIEW

File: admin_manual/configuration_server/occ_command.rst [hunk 1]
```diff
@@ -1203,7 +1203,7 @@ Trashbin
 
  trashbin
   trashbin:cleanup  [--all-users] [--] [<user_id>...]  Permanently remove deleted files
-  trashbin:restore  [--all-users] [--] [<user_id>...]  Restore deleted files
+  trashbin:restore  [--all-users] [--scope[=SCOPE]] [--restore-from[=RESTORE-FROM]] [--restore-to[=RESTORE-TO]] [--dry-run] [--] [<user_id>...]  Restore all deleted files according to the given filters
```
Comment (come-nc) [inline admin_manual/configuration_server/occ_command.rst:1206]:
```suggestion
  trashbin:restore  [--all-users] [--scope[=SCOPE]] [--restore-from[=RESTORE-FROM]] [--restore-to[=RESTORE-TO]] [--dry-run] [--] [<user_id>...]  Restore deleted files according to the given filters
```

**Review by come-nc — APPROVED** [review]

File: admin_manual/configuration_server/occ_command.rst (hunk 1 above)
Comment (R0Wi) [reply admin_manual/configuration_server/occ_command.rst:1206]: Good catch 😄

**Comment by R0Wi** [comment]
Updated docs and force-pushed to reflect latest code changes

**Pull request merged**